src/framework/filterControl/filterDropdown/_filterFormDropdown.scss

```diff
@@ -0,0 +1,43 @@
+
+.filterFormDropdown {
+  position: relative;
+}
+
+  .filterFormDropdown__form {
+    background: white;
+    border-radius: $addedFilter_borderRadius;
+    border: 1px solid $addedFilter_borderColor;
+    color: #474b4e;
+    font-size: $addedFilter_fontSize;
+    position: absolute;
+    top: 0;
+    z-index: 1;
+  }
+
+    .filterFormDropdown__form__header {
+      background: $componentBackgroundColor;
+      border-bottom: 1px solid #d4d8db;
```

Comment:
Okay, if you cannot find it then may be it can be introduced if you think it is commonly occurring. Shall adjust later, please ignore for now.